runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java

```diff
@@ -27,45 +27,106 @@
 
 import com.google.common.eventbus.Subscribe;
 import javax.inject.Inject;
+
+import com.google.inject.Provides;
```

Comment:
This should be sorted above `javax.inject.Inject` and below `com.google.common.eventbus.Subscribe`